rero_ils/modules/libraries/jsonschemas/libraries/library-v0.0.1.json

```diff
@@ -439,6 +439,12 @@
             },
             "address": {
               "$ref": "#/definitions/address_block"
+            },
+            "extra": {
+              "title": "Extra information",
+              "description": "VAT number, note, ...",
```

Comment:
Possibly need to mention that this field is used only in the notifications  and not somewhere else. but not really sure.